tests/api/test_web_auth_hardening.py

```
@@ -160,7 +251,9 @@ def test_chat_stream_websocket_rejects_query_param_api_key(
     assert exc_info.value.code == 1008
 
 
-def test_default_cors_origins_do_not_use_wildcard(monkeypatch: pytest.MonkeyPatch) -> None:
+def test_default_cors_origins_do_not_use_wildcard(
+    monkeypatch: pytest.MonkeyPatch,
```

Ruff (0.15.10)

[warning] Unused function argument: `monkeypatch`

(ARG001)